src/Nancy.Serialization.ServiceStack/Helpers.cs

```diff
@@ -23,10 +23,10 @@ public static bool IsJsonType(string contentType)
 
             var contentMimeType = contentType.Split(';')[0];
 
-            return contentMimeType.Equals("application/json", StringComparison.InvariantCultureIgnoreCase) ||
-                   contentMimeType.Equals("text/json", StringComparison.InvariantCultureIgnoreCase) ||
-                  (contentMimeType.StartsWith("application/vnd", StringComparison.InvariantCultureIgnoreCase) &&
-                   contentMimeType.EndsWith("+json", StringComparison.InvariantCultureIgnoreCase));
+            return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
+                   contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
+                  (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&

```

Comment:
Ideally, we'd like this check to be the same as the default serializer; https://github.com/NancyFx/Nancy/blob/master/src/Nancy/Responses/DefaultJsonSerializer.cs#L100-L102